index.native.js

```diff
@@ -97,6 +110,26 @@ const TinderCard = React.forwardRef(
     }))
     settings.swipeThreshold = swipeThreshold
 
+    const getBackAnimationType = () => {
+      if (backAnimationType === 'standard') {
+        return physics.animateBack;
+      } else if (backAnimationType === 'slow') {
+        return physics.animateSlow;
+      } 
+      return physics.none;
```

Comment:
This could be more readable.

```ts
      if (backAnimationType === 'standard') return physics.animateBack;
      if (backAnimationType === 'slow') return physics.animateSlow;
      return physics.none;
```